setup.py

```diff
@@ -0,0 +1,22 @@
+from distutils.core import setup
+
+import numpy as np
+import setuptools
+
+from Cython.Build import cythonize
+
+
+extensions = [setuptools.Extension(
+    "mo_pack._packing",
+    ["lib/mo_pack/_packing.pyx"],
```

Comment:
It's a good job you waited for tests, I found that the name of the source (the .pyx file) and the name of the module (in this case _packing) have to match. Otherwise, when trying to import mo_pack, you get the error "ImportError: dynamic module does not define init function"